gatsby-config.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
         name: "Apply",
         route: "/apply",
       },
+      {
+        name: "Clients",
```

Comment:
we should also include this page in the footer links

Reply:
yes, I already merged a PR to sync up the footer links with the nav links just haven't merged from master yet.